### DISCUSSION TITLE
Where is the current location for javadoc ?

### DISCUSSION THREAD

**Discussion by mores** [Netflix/dgs-framework#2089] [General]
I am reading this page: https://netflix.github.io/dgs/advanced/java-client/

It says "Refer to the GraphQLResponse JavaDoc for the complete list of supported methods."

I found: https://javadoc.io/doc/com.netflix.graphql.dgs/graphql-dgs/latest/index.html but it looks to be incomplete.

**Comment by paulbakker** [comment]
We don't publish Javadocs, but it's in the source code, so your IDE will be
the best place to look.

**Reply by mores** [reply]
I cloned the repo, then ran ./gradlew javadoc

It created javadocs for 8 directories, graphql-dgs-client was not one of them:
`-rw-rw-r-- 1 smore smore 30263 Dec 18 09:43 graphql-dgs/build/docs/javadoc/index-all.html`
`-rw-rw-r-- 1 smore smore 56455 Dec 18 09:44 graphql-dgs-example-shared/build/docs/javadoc/index-all.html`
`-rw-rw-r-- 1 smore smore  3704 Dec 18 09:44 graphql-dgs-extended-validation/build/docs/javadoc/index-all.html`
`-rw-rw-r-- 1 smore smore 12151 Dec 18 09:44 graphql-dgs-reactive/build/docs/javadoc/index-all.html`
`-rw-rw-r-- 1 smore smore 19023 Dec 18 09:44 graphql-dgs-spring-graphql-example-java/build/docs/javadoc/index-all.html`
`-rw-rw-r-- 1 smore smore 14659 Dec 18 09:44 graphql-dgs-spring-graphql-example-java-webflux/build/docs/javadoc/index-all.html`
`-rw-rw-r-- 1 smore smore  3493 Dec 18 09:43 graphql-dgs-spring-graphql-test/build/docs/javadoc/index-all.html`
`-rw-rw-r-- 1 smore smore 32277 Dec 18 09:43 graphql-error-types/build/docs/javadoc/index-all.html`
